deploy/heapster-controller.yaml

```diff
@@ -18,6 +18,15 @@ desiredState:
               - 
                 name: "INFLUXDB_HOST"
                 value: "monitoring-influxdb"
+            volumeMounts:
+              - name: ssl-certs
+                mountPath: /etc/ssl/certs
```

Comment:
Noob question: Is this the default location for storing certs for most applications?